terraform/outputs.tf

```diff
@@ -0,0 +1,17 @@
+output "app_name" {
+  value = juju_application.pvcviewer_operator.name
+}
+
+output "provides" {
+  value = {
+    metrics_endpoint  = "metrics-endpoint",
+    grafana_dashboard = "grafana-dashboard"
+  }
+}
+
+output "requires" {
+  value = {
+    object_storage = "object-storage",
+    logging        = "logging"
+  }
+}
```

Comment:
This should expose the endpoints of this specific charm and there is no [object_storage endpoint](https://github.com/canonical/pvcviewer-operator/blob/main/metadata.yaml#L22-L25).